privaterelay/templates/includes/vpn-promo-banner.html

```diff
@@ -0,0 +1,17 @@
+{% load static %}
+{% load relay_tags %}
+{% load ftl %}
+{% ftlconf bundle='privaterelay.ftl_bundles.main' %}
+
+<div id="vpnPromoBanner" class="vpn-promo-banner closed">
+	<div class="vpn-promo-banner-wrapper">
+		<img alt="Mozilla VPN" class="vpn-promo-logo" src="{% static 'images/logos/mozilla-vpn.svg' %}"/>
+		<div class="vpn-promo-copy">
+			<strong>{% ftlmsg 'vpn-promo-price-headline' %}</strong>
+			<span>{% ftlmsg 'vpn-promo-price-copy' %}</span>
+			<small>{% ftlmsg 'vpn-promo-price-disclaimer' %}</small>
+		</div>
+		<a class="vpn-promo-cta" href="https://www.mozilla.org/products/vpn/?utm_source=monitor&utm_medium=monitor&utm_campaign=intro-pricing">{% ftlmsg 'vpn-promo-cta' %}</a>
```

Comment:
Can we add a `sendGaPing` call here too, so we can see how many times the CTA is clicked all on the Relay side of GA?